src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -537,33 +544,27 @@ def run(self, timeout):
     return result.stats
 
 
-def record_cross_pollination_stats(
-    pruner_stats, pollinator_stats, project_qualified_name, sources, tag,
-    initial_corpus_size, minimized_corpus_size_units, method):
+def _record_cross_pollination_stats(stats):
   """Log stats about cross pollination in BigQuery."""
-  # TODO(mpherman): Find a way to collect these stats for OSS Fuzz.
-  if environment.is_untrusted_worker():
+  if not stats:
```

Comment:
is this expected to happen regularly? maybe we should log an error or a warning at least?

Reply:
This will happen when corpus pruning timeouts or has no corpus and stats could not be collected. These will have already been logged. I added a comment to clarify this case.